directory.json

```diff
@@ -0,0 +1,17 @@
+
+{
+	"revere": {
```

Comment:
Eventually, would this data be pulled from the spreadsheet backing the form?
https://docs.google.com/forms/d/e/1FAIpQLSciFMu6YT1xqiamqetzs2Qcw-I_mVTR_n687D-ZgAMaw07Vug/viewform

Reply:
ya. Imma delete this file. It was more there as a proof of concept